docs/developer/payments/lifecycle.mdx

```diff
@@ -359,6 +359,8 @@ Each event can be excluded from calculation if it has matching `*_FAILURE` event
 
 To learn more about each event type, see [Transaction event types](#transaction-event-types) section.
 
+To learn more about amount calculations see [Price calculations](/developer/payments/price-calculations) page.
```

Comment:
We need `.mdx` ending so that file renaming works correctly 🤓

```suggestion
To learn more about amount calculations see [Price calculations](/developer/payments/price-calculations.mdx) page.
```